haicrypt/hcrypt_tx.c

```diff
@@ -149,11 +144,6 @@ int HaiCrypt_Tx_Process(HaiCrypt_Handle hhc,
 	/* Manage Key Material (refresh, announce, decommission) */
 	hcryptCtx_Tx_ManageKM(crypto);
 
```

Comment:
@jeandube : Note that I'm taking it as a good deal that the call to `hcryptCtx_Tx_ManageKM` doesn't modify the assignment of `crypto->ctx`; as far as I could check it, it shouldn't.